static/css/login.scss

```diff
@@ -1,30 +1,134 @@
-@import "variables";
-
-body {
-  display: flex;
-  justify-content: center;
-  background-color: $lighter-grey;
-
-  .login-container {
-    // background-color: $lighter-grey;
-    width: 700px;
-    margin-top: 100px;
-
-    form {
-      display: flex;
-      flex-direction: column; 
-      
-      input {
-        margin-top: 20px;
-        padding: 8px;
-        border-radius: 6px;
-        border: 1px black solid;
-
-        &:focus {
-          outline: none;
-          border: 2px solid $blue;
-        }
-      }
-    }
-  }
+@import url(https://fonts.googleapis.com/css?family=Exo:100,200,400);
+@import url(https://fonts.googleapis.com/css?family=Source+Sans+Pro:700,400,300);
+@import url('https://fonts.googleapis.com/css2?family=Chango&display=swap');
+
+body{
+	margin: 0;
+	padding: 0;
+	background: #fff;
+
+	color: #fff;
+	font-family: Arial;
+	font-size: 12px;
+}
+
+.body{
+	position: absolute;
+	top: -20px;
+	left: -20px;
+	right: -40px;
+	bottom: -40px;
+	width: auto;
+	height: auto;
+	background-image: url(https://dvzo.ch/wp-content/uploads/2019/04/B%C3%A4retswil-1969-HPB-II.jpg);
+	background-size: cover;
+	-webkit-filter: blur(2px);
+	z-index: 0;
+}
+
+.grad{
+	position: absolute;
+	top: -20px;
+	left: -20px;
+	right: -40px;
+	bottom: -40px;
+	width: auto;
+	height: auto;
+	background: -webkit-gradient(linear, left top, left bottom, color-stop(0%,rgba(0,0,0,0)), color-stop(100%,rgba(0,0,0,0.65))); /* Chrome,Safari4+ */
+	z-index: 1;
+	opacity: 0.7;
+}
+
+.header{
+	color: #fff;
+	font-family: 'Chango', sans-serif;
+	font-size: 42px;
+	font-weight: 200;
+	z-index: 2;
+}
+
+div.header span{
+	color: #ed174b !important;
+}
+
+.login{
+  position: absolute;
+	top: calc(50% - 75px);
+	left: calc(50% - 135px);
+	height: 150px;
+	width: 350px;
+	padding: 10px;
+	z-index: 3;
+}
+
+.login input[type=text]{
```

Comment:
Are these styles uniquely for the login form styling? Otherwise they should not be in this file...